src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -30,4 +31,9 @@ case class Account(
 
   def withStorage(storageRoot: ByteString): Account =
     copy(storageRoot = storageRoot)
+
+  override def toString: String =
+    s"Account(nonce: $nonce, balance: $balance, " +
+      s"storageRoot: ${Hex.toHexString(storageRoot.toArray[Byte])}, codeHash: ${Hex.toHexString(codeHash.toArray[Byte])})"
```

Comment:
(Refactor is coming post milestone 2, Hex.toHexStting can be implicit value class, ByteString might become scorex implementation of same ...)